src/java.base/share/classes/java/lang/Thread.java

```diff
@@ -1867,10 +1874,6 @@ public final boolean isAlive() {
      * This method is non-final so it can be overridden.
      */
     boolean alive() {
-        return isAlive0();
-    }
-
-    private boolean isAlive0() {
         synchronized (interruptLock) {
             return eetop != NO_REF;
```

Comment:
There's no longer any value in using `synrchronized` here now that `eetop` is `volatile`.

Reply:
Removed synchronization on `interruptLock` for `alive()`.